tf2_pykdl_ros/src/tf2_pykdl_ros/tf2_pykdl_ros.py

```diff
@@ -104,10 +104,10 @@ def to_msg_frame(frame: FrameStamped) -> PoseStamped:
     msg.pose.position.y = v[1]
     msg.pose.position.z = v[2]
     q = frame.frame.M.GetQuaternion()
-    msg.pose.orientation.x = q.x
-    msg.pose.orientation.y = q.y
-    msg.pose.orientation.z = q.z
-    msg.pose.orientation.w = q.w
```

Comment:
If this method of calling the elements of a quaternion is still suported then I would prefer this as its more clear

Reply:
No, this is not supported, as the return type is just a simple tuple

Reply:
Did you confirm that the order of this tuple is indeed x,y,z,w?

Reply:
Yes,
https://github.com/orocos/orocos_kinematics_dynamics/blob/db25b7e480e068df068232064f2443b8d52a83c7/python_orocos_kdl/PyKDL/frames.cpp#L358